src/components/AudioWaveform.module.css

```
@@ -0,0 +1,11 @@
+.waveform {
+  display: block;
+  overflow: hidden;
+  color: var(--audio-waveform-color, rgba(52, 103, 188, 0.65));
+}
+
+.waveform path {
+  fill: none;
+  stroke: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)